apps/challenges/views.py

```diff
@@ -777,6 +839,87 @@ def create_challenge_using_zip_file(request, challenge_host_team_pk):
         response_data = {"error": message}
         return Response(response_data, status=status.HTTP_406_NOT_ACCEPTABLE)
 
+    # Manipulating the challenge data of the template challenge with data sent by hosts
```

Comment:
didn't get what you meant by manipulating? are you just overriding the template config values using the values given by host? if yes, can you change the comment